e2-studio-star-rx72n-firmware/tests/test_bms_monitoring_task.c

```
@@ -173,63 +251,187 @@ void test_bms_task_reads_battery_data(void)
+ * @details
+ * Verifies that bms_monitor_task_create() accepts non-default threshold
+ * values when they satisfy the constraint soc_critical_pct < soc_warning_pct.
+ */
+void test_bms_task_create_custom_thresholds(void)
+{
+  rx_err_t err;
+
+  mock_tx_set_thread_create_return(TX_SUCCESS);
+
+  /* Custom thresholds: 30% warning, 10% critical */
+  const bms_monitor_config_t custom_config = {
```

Clang (14.0.6)

[warning] variable 'custom_config' is not initialized

(cppcoreguidelines-init-variables)